src/page-modules/departures/details/decoration-line/decoration-line.module.css

```diff
@@ -0,0 +1,35 @@
+.decoration {
```

Comment:
The decoration line should be right between the left column and the right column. As it is now the right column is closer.

<img width="246" alt="image" src="https://github.com/AtB-AS/planner-web/assets/14045515/f7f63bf8-8d21-478f-8bb9-e9677b908915">

Easily solved with a `transform: translateX(-50%)`, possibly some other easy solution as well.

Becomes like this:

<img width="254" alt="image" src="https://github.com/AtB-AS/planner-web/assets/14045515/9e5de717-da9c-476a-986b-09142a8f9d32">